README.md

```diff
@@ -7,66 +7,77 @@
 
 ### Java
 To run JRuby you will need a JRE (the JVM runtime environment) version 7 or higher.
-```
-$ java --version
-  java 9
-  Java(TM) SE Runtime Environment (build 9+181)
-  Java HotSpot(TM) 64-Bit Server VM (build 9+181, mixed mode)
-```
+
+    $ java --version
+      java 9
+      Java(TM) SE Runtime Environment (build 9+181)
+      Java HotSpot(TM) 64-Bit Server VM (build 9+181, mixed mode)
 
 ### Ruby
-Follow these instructions to [install JRuby](https://github.com/psu-libraries/psulib_traject/wiki/Install-JRuby) if you do not already have it.
-```
-$ ruby --version
-  jruby 9.2.11.1
-```
+Follow these instructions to [install JRuby](https://github.com/psu-libraries/psulib_traject/wiki/Install-JRuby) if you
+do not already have it.
+
+    $ ruby --version
+      jruby 9.2.11.1
 
 ## Development Setup
 
-1.  [Make sure you have ssh keys established on your machine](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/#generating-a-new-ssh-key)
-1.  [Make sure you have docker installed and running](https://docs.docker.com/install/)
-1.  Clone the application and install.
-    ``` 
+[Make sure you have ssh keys established on your machine](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/#generating-a-new-ssh-key)
+
+[Make sure you have docker installed and running](https://docs.docker.com/install/)
+
+Clone the application and install:
+
     $ git clone git@github.com:psu-libraries/psulib_traject.git
     $ cd psulib_traject
     $ bundle install
-    ```
-   
-## Build an Index
 
-1. Solr config files need to be copied from [psulib_blacklight](https://github.com/psu-libraries/psulib_blacklight/tree/master/solr/conf):
+## Configuration
+
+For local development, you can change the settings by adding configuration files. These will be ignored by git.
+
+### Solr
+
+Create 2 files: `config/settings.local.yml` and `config/test/test.local.yml` and add the following lines to each:
     
-    ```
-    $ bundle exec rake solr:conf
-    ```
-   
-1. Start Solr
+    solr:
+      url: http://localhost:8983/solr/
+      port: 8983
+
+Change the URL and port numbers if you want to use a different port.
+
+### Traject
 
-    If Docker Solr isn't running (check `docker ps`) run
+When using jruby, traject will use multiple threads, but we want to tailor that to our system. In
+`config/settings.local.yml` add:
 
-    ```
-    $ bundle exec rake solr:up
-    ```
+    hathi_overlap_path: spec/fixtures/hathitrust/overlap.tsv
+    processing_thread_pool: 5
+   
+## Build an Index
+
+Start Solr via the Docker container
+    
+    $ bundle exec rake docker:first_start
```

Comment:
@awead should this be `bundle exec rake solr:up` maybe 🤔

Reply:
Also, I could get `docker:up` to work by only switching back to backticks rather than `exec`, wondering if I am missing sth, does `docker:up` works for you?

Reply:
@banukutlu try it again. I've simplified it a bit.